api/src/test/java/uk/gov/dhsc/htbhf/claimant/reporting/payload/ReportPropertiesFactoryTest.java

```diff
@@ -205,6 +244,27 @@ private ReportClaimMessageContext aReportClaimMessageContext(LocalDateTime times
                 .build();
     }
 
+    private ReportPaymentMessageContext aReportPaymentMessageContext(LocalDateTime timestamp,
+                                                                     List<LocalDate> datesOfBirthOfChildren,
+                                                                     LocalDate expectedDeliveryDate) {
+        Claim claim = createClaimWithDueDateAndPostcodeData(expectedDeliveryDate);
+        return ReportPaymentMessageContext.builder()
+                .paymentAction(SCHEDULED_PAYMENT)
+                .paymentCycle(Optional.of(aPaymentCycleWithClaim(claim)))
+                .claim(claim)
+                .datesOfBirthOfChildren(datesOfBirthOfChildren)
+                .timestamp(timestamp)
+                .build();
+    }
+
+    private Claim createClaimWithDueDateAndPostcodeData(LocalDate expectedDeliveryDate) {
```

Comment:
Worth adding to the ClaimTestDataFactory? If nothing else it would make this test class smaller.

Reply:
Moved into test factory.